### PR TITLE
add script to remove styles from raw HTML files

### PR DESCRIPTION
**Pull request by alifeee**
https://gist.github.com/alifeee/75df204c2be3815806e1d4be47e54d08

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,7 @@ <h3><a id="tools" href="#tools">Tools and plugins</a></h3>
             <li>Drupal users, <a href="https://www.drupal.org/project/naked">get the module</a>!</li>
             <li>If you are game you can use <a href="https://weblog.200ok.com.au/">Ben Buchanan</a>’s <a href="https://bitbucket.org/snippets/200ok/dKpk">JavaScript snippet</a>. Works anywhere, removes everything, buyer beware!</li>
             <li>If you are a Rails developer, check out the <a href="https://github.com/danielsellergren/css_naked_day_rails">css_naked_day_rails</a> gem by <a href="https://danielsellergren.com/">Daniel Sellergren</a>.</li>
+            <li>Use <a href="https://gist.github.com/alifeee/75df204c2be3815806e1d4be47e54d08">this bash script</a> to remove stylesheets, style tags, and inline styles from multiple raw HTML or other files.</li>
```
Comment (j9t) [inline index.html:118]: Please use a similar wording as the other points, which start with a person or a condition.

Also, “`style` elements” would be most precise.

For “stylesheets,” I suggest the spelling “style sheets,” but I don’t think we established a convention on the site. (@arkhi, any views?)

**Review by j9t — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (alifeee) [reply index.html]: I have changed as per your suggestions 👍

File: index.html [hunk 2]
```diff
@@ -115,6 +115,7 @@ <h3><a id="tools" href="#tools">Tools and plugins</a></h3>
             <li>Drupal users, <a href="https://www.drupal.org/project/naked">get the module</a>!</li>
             <li>If you are game you can use <a href="https://weblog.200ok.com.au/">Ben Buchanan</a>’s <a href="https://bitbucket.org/snippets/200ok/dKpk">JavaScript snippet</a>. Works anywhere, removes everything, buyer beware!</li>
             <li>If you are a Rails developer, check out the <a href="https://github.com/danielsellergren/css_naked_day_rails">css_naked_day_rails</a> gem by <a href="https://danielsellergren.com/">Daniel Sellergren</a>.</li>
+            <li>If you have lots of raw HTML, use <a href="https://gist.github.com/alifeee/75df204c2be3815806e1d4be47e54d08">this bash script</a> to remove style sheets, <code>style</code> tags, and inline styles.</li>
```
Comment (j9t) [inline index.html:118]: Thanks! One last thing—this seems to mean elements, [not tags](https://annevankesteren.nl/2004/05/tags-vs-elements).

Comment (alifeee) [reply index.html]: ah yes. I meant to change that before, but I got distracted with \<code\>. Thanks for catching again.

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
Thank you!

**Pull request merged**